docs/pmm-upgrade/migrating_from_pmm_2.md

```diff
@@ -13,45 +13,100 @@ Before upgrading to PMM 3, ensure your PMM 2 Server is running the latest versio
 
 ## Step 2: Migrate PMM 2 Server to PMM 3
 
-Follow these manual steps to upgrade your PMM 2 Server to PMM 3:
-{.power-number}
+=== "For Docker Volume"
```

Comment:
I'd rename it to something like "For those running PMM v2 with docker volumes" to clarify that these steps should be executed if PMM2 server is running using docker volume container